pwndbg/lib/memory.py

```diff
@@ -134,6 +132,11 @@ def permstr(self) -> str:
         )
 
     def __str__(self) -> str:
+        # This module requires GDB, so it causes import failures in unit tests.
+        # This will stop being a problem as soon as this module gets ported to
+        # aglib.arch, but, for now we have to add this as a quick stopgap.
+        import pwndbg.gdblib.arch
```

Comment:
I don't like this but ok

Reply:
This should get fixed in the next one or two PRs.